packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -571,7 +571,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         type: ContainerMessageType,
         content: any,
         localOpMetadata: unknown,
-        opMetaData: unknown,
+        opMetaData: Record<string, unknown> | undefined,
```

Comment:
Constraining 'opMetadata' to types that we can merge (i.e., can't be a number, a boolean, etc.)